src/network.cpp

```diff
@@ -43,6 +56,20 @@ namespace oxen::quic
         close();
     }
 
+	std::shared_ptr<Endpoint> Network::endpoint(const Address &local_addr)
+	{
+		return endpoint(const_cast<Address&>(local_addr));
```

Comment:
This cast is dangerous, but also seems unnecessary; why do we ever need to take the address as a non-const reference?